machine/wnc_x86_64/wnc.patch/installer/x86_64/install-arch

```diff
@@ -0,0 +1,499 @@
+# x86_64 specific ONIE installer functions
```

Comment:
All of these patches in wnc_x86_64/wnc.patch/* will be unacceptable as is.

If some functionality is lacking (or unclear how to do something) please ask questions on the mailing list.  Perhaps something is not clear, or you have a valid point and we need to make ONIE better.